src/services/variorum/Variorum.cpp

```diff
@@ -34,9 +34,12 @@ namespace
 std::tuple<bool, uint64_t> measure(const std::string& name)
 {
     double power_watts;
-    json_t *power_obj = json_object();
+    json_t *power_obj = NULL;
+    char *s = NULL;
 
-    int ret = variorum_get_node_power_json(power_obj);
+    s = (char *) malloc(800 * sizeof(char));
```

Comment:
@slabasan I think you need a `free(s)` and a `json_decref(power_obj)`  to match the `malloc` and creation of the `json_t` variable in `Variorum.cpp` (at line 56 below should be a good place for it I think?).

Reply:
Nice catch, thank you!